
=====================================================================
This module is intended solely for building or source activating user
python environments, i.e.,

    mamba create -n myenv -c conda-forge

or

    source activate myenv

To list available environments, run:

    mamba info --envs

See our docs: https://links.asu.edu/solpy

Any other use is NOT TESTED.
=====================================================================

  
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/surfaces.py:60: UserWarning: Geodesic distance module is unavailable; some functionality for surfaces will be unavailable.
  warnings.warn(msg)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
/home/dhughe13/.conda/envs/tvb_env/lib/python3.12/site-packages/tvb/datatypes/connectivity.py:321: RuntimeWarning: invalid value encountered in cast
  self.idelays = numpy.rint(self.delays / dt).astype(numpy.int32)
slurmstepd: error: *** JOB 22410354 ON sc065 CANCELLED AT 2025-01-15T14:22:19 ***
